src/scorepy/cInstrumenter.cpp

```diff
@@ -114,7 +114,11 @@ bool CInstrumenter::on_event(PyFrameObject& frame, int what, PyObject*)
     {
     case PyTrace_CALL:
     {
+#if PY_VERSION_HEX < 0x030b00a0 // python 3.11.0
```

Comment:
`PyFrame_GetCode` was introduced in 3.9

Reply:
> Part of the [Stable ABI](https://docs.python.org/3/c-api/stable.html#stable) since version 3.10.
Not sure if we should use it with in 3.9 or 3.10. 

Since `PyFrame_GetGlobals` is new in 3.11 I thought it is cleaner to do only make two buckets: starting with 3.11 and before 3.11